meetings/minutes/2020-08-03-TSC-Minutes.md

```diff
@@ -0,0 +1,216 @@
+# Minutes from the 2020-08-03 Laminas Technical Steering Committee Meeting
+
+- When: 2020-08-03 at 19:00 UTC, until 20:39 UTC
+- Where: Laminas Slack #tsc-meeting channel
+- Attending:
+  - Abdul Malik Ikhsan
+  - Aleksei Khudiakov
+  - Andreas Heigl
+  - Frank Brückner
+  - Geert Eltink
+  - James Titcumb
+  - Luís Cobucci
+  - Marco Pivetta
+  - Max Bösing
+  - Michał Bundyra
+  - Matthew Setter
+  - Matthew Weier O'Phinney
+  - Rob Allen
+
+Quorum was met (13 out of 15 members were present).
+
+## Agenda
+
+- New branch model on security-only repositories?
+  ([jump to discussion](#apply-new-branching-model-to-security-only-components))
+
+  As we are about to mark packages to receive security-only fixes, @boesing
+  would like to discuss if we want to apply the new branch model (related to the
+  new release-process) to those repositories aswell or if we exclude these?
```

Comment:
google is saying that `aswell` is:
1. Archaic form of `as well`, or
2. Misspelling of `as well`